website/public/index.ejs.html

```diff
@@ -207,6 +207,7 @@ <h2 id="linter-feature-comparison">Features</h2>
               <th><a href="https://rslint.org/">RSLint</a></th>
               <th><a href="https://deno.land/">Deno</a></th>
               <th><a href="https://rome.tools/">Rome</a></th>
+              <th><a href="https://sonarlint.org/">SonarLint</a></th>
```

Comment:
The table looks really wide. We could use `&shy;` to hyphenate "SonarLint" as "Sonar-Lint" to reduce the table width:
```suggestion
              <th><a href="https://sonarlint.org/">Sonar&shy;Lint</a></th>
```